go/test/endtoend/cluster/cluster_process.go

```diff
@@ -765,19 +765,18 @@ func (cluster *LocalProcessCluster) populateVersionInfo() error {
 	return err
 }
 
+var versionRegex = regexp.MustCompile(`Version: ([0-9]+)\.([0-9]+)\.([0-9]+)`)
```

Comment:
any reason why moving this as a global?

Reply:
This is just the usual pattern I've seen: init the regex once and reuse it for the lifetime of the program, rather than reparsing on each call.

Happy to revert if you'd like.

Reply:
I think this makes sense to move :+1:.